Builds/Game/Config/network.json

```diff
@@ -0,0 +1,8 @@
+{
+    "_preload": true,
+    "_type": 2,
+    "NetworkType": 2,
+    "Ckey": "editorUser",
+    "ServerAddress": "127.0.0.1",
+    "ServerPort": 2158
```

Comment:
Port # here is different from that in the .bat files. Intentional?

Reply:
Nope ! good catch. Will fix tomorrow.